cypress/integration/cookieBannerWS.js

```diff
@@ -1,7 +1,7 @@
 import config from '../support/cookieConfig';
 import { worldServiceCookieBannerTranslations } from '../support/bodyTestHelper';
 import describeForEuOnly from '../support/describeForEuOnly';
-import describeForLocalOnly from '../support/describeForLocalOnly';
+import describeForLocalOnly from '../support/limitEnvRuns';
```

Comment:
All of these default imports from the `limitEnvRuns` file should be replaced with destructured imports
```suggestion
import { describeForLocalOnly } from '../support/limitEnvRuns';
```